api/urls.py

```diff
@@ -35,9 +35,9 @@ def to_url(self, value):
 )
 
 urlpatterns = [
-    path('v1/premium_countries',
-         premium_countries,
-         name='premium_countries'
+    path('v1/runtime_data',
```

Comment:
I renamed this API to no longer return just data about which country the user is in and which plan is applicable to them, but also the values of some environment variables. This allows us to compile the front-end once, and have it adapt to the environment it's deployed to, as requested by Aaron.

Reply:
Wouldn't a change like this require a `v2` upgrade?

Reply:
Actually, I checked to be sure, and the add-on is already using it. So I restored the `premium_countries` API in 64cbd90d.